cli/auth_account_command.go

```diff
@@ -117,14 +117,14 @@ func configureAuthAccountCommand(auth commandHost) {
 		f.Flag("jetstream", "Enables JetStream").Default("false").IsSetByUser(&c.jetStreamIsSet).BoolVar(&c.jetStream)
 		f.Flag("js-consumers", "Sets the maximum Consumers the account can have").Default("-1").IsSetByUser(&c.maxConsumersIsSet).Int64Var(&c.maxConsumers)
 		f.Flag("js-disk", "Sets a Disk Storage quota").PlaceHolder("BYTES").StringVar(&c.storeMaxString)
-		f.Flag("js-disk-stream", "Sets the maximum size a Disk Storage stream may be").PlaceHolder("BYTES").Default("-1").StringVar(&c.storeMaxStreamString)
+		f.Flag("js-disk-stream", "Sets the maximum size a Disk Storage stream may be").Default("-1").StringVar(&c.storeMaxStreamString)
```

Comment:
I dont think universally just updating all of these with a script is the right approach, in the previous one where the default is something useful like `1w` it makes sense to show the default, here `-1` is more an implementation detail than a useful default users should know about so the placement string should be used in stead.

We will need to evaluate each case and decide if it makes sense to show the default.

A lot of this code was written before fisk has IsSetByUser() and so some of these defaults are there only to compensate for the lack of that feature rather than a default thats useful to the user.

Reply:
@ripienaar would you be open to a separate PR, where places that use "meaningless" defaults are update to IsSetByUser ?

Reply:
Yes for now let’s drop placeholders only where it make sense. Later we can move to IsSetByUser()